src/main/java/com/example/ludogoriesoft/lukeriaerpapi/models/Package.java

```diff
@@ -20,15 +18,20 @@ public class Package {
     private String name;
 
     @Column(name = "available_quantity")
+    @Min(value = 1, message = "Наличните бройки не могат да бъдат отрицателно число!")
```

Comment:
It's more accurate to say:
"Наличените бройки трябва да бъдат положително число!"/"Наличените бройки трябва да бъдат по-големи от 0!"

Otherwise, it means that it can be 0 which is not true.